docs/actions/match.md

```diff
@@ -161,6 +161,23 @@ end
 
 Then all your team has to do is `fastlane certificates` and keys, certs and profiles for all targets will be synced.
 
+#### Handle multiple apps per developer/distribution certificate
+
+If you want to use single developer and distribution certificate for multiple apps belonging to the same development team, you may use the same signing identities repo and branch to store signing identities for your apps:
```

Comment:
... use single ... => ... use a single ...
repo => repository
... store signing ... => ... store the signing ...

Reply:
Done